.gitignore

```diff
@@ -214,3 +214,5 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
+
+Output/
```

Comment:
Is there a reason we're using `Output/PowerShellLogging` rather than building to a versioned folder (given there was no build script previously, it seems this would assist with the workflow we've previously discussed).